compiler/circle-opselector-test/README.md

```diff
@@ -0,0 +1,52 @@
+# Circle-OpSelector-test
+
+## Usage
+
+- This test didn't consider **if** and **while** nodes yet. 
+
+**Single file test**
+
+- `tflite-file-name` must be contained in [tflite receipes](https://github.com/Samsung/ONE/tree/master/res/TensorFlowLiteRecipes)
```

Comment:
```suggestion
- `tflite-file-name` must be contained in [tflite receipes](https://github.com/Samsung/ONE/tree/5e53631d84a05467547296dd2f948b8a7e907161/res/TensorFlowLiteRecipes)
```

Please do not use `master` url because it can be changed.